fsw/pc-rtems/README_RTEMS_5.txt

```diff
@@ -37,7 +37,7 @@ cd rtems-source-builder/rtems
 Note - at the time of this writing 5.1 is the current "stable" branch and
 5.1 represents the latest point release tag on that branch. We are going to
 use the "5" branch, which will eventually be released as 5.2. The latest
-5 branch has a bux fix for the RFS file system needed by the cFS for this target.
+5 branch has a bug fix for the RFS file system needed by the cFS for this target.
 If you use the 5.1 release tag (or tarball) you will most likely not be able
 to run the cFS without an error.
 
```

Comment:
Remove extra space between for and fully in `When using QEMU this allows for fully scripted testing.` on line 262.